core/__tests__/actions/setupSteps.ts

```diff
@@ -118,35 +118,28 @@ describe("actions/setupSteps", () => {
       expect(destinationStep.disabled).toEqual(true);
     });
 
-    test.each(["source", "group", "schedule", "destination"])(
-      "%s setupSteps href changes based on modelId",
-      async (topic) => {
-        connection.params = { csrfToken, modelId: null };
-        const { setupSteps: setupStepsNoModelId } =
-          await specHelper.runAction<SetupStepsList>(
-            "setupSteps:list",
-            connection
-          );
-        let destinationStep = setupStepsNoModelId.find(
-          (s) => s.key === `create_a_${topic}`
+    test("destination setupSteps href changes based on modelId", async () => {
```

Comment:
Wouldn't the "Configure Your Model" step also be by model id?

Reply:
Test added in https://github.com/grouparoo/grouparoo/pull/2773/commits/5f9a6541528e5b65ae9cf55f128b8250fb0cc38a